finance.md

```diff
@@ -0,0 +1,69 @@
+
+Our main financial institution is **Desjardins** and our main funds are stored in a business chequing account, where all outgoing transactions require 2 signatures from Members of the Finance WG.
+
+Hypha is expected to conduct business with clients from other countries and is ready to [accept funds in CAD, USD, and EUR](https://transferwise.com/help-smart/11/getting-started/2571907/what-currencies-do-you-support). We use **TransferWise** to minimize transaction and currency exchange fees. Our Transferwise account has local accounts in Canada, United States, and Europe, so we can bill clients as a local financial institution in their local currency. Our TransferWise account does not require multiple signatures, and the Finance WG is responsible for transfering money into our main Desjardins account when accumulated funds exceed a CAD 10,000 threshold.
+
+## Invoicing a client
+
+Hypha charges clients on a sliding scale, and projects may be billed hourly, weekly, monthly, or per project, depending on the specifics of each project. For all projects, regardless of how a customer is charged, Members log their hours using **Clockify**. For clients and billing models that require timesheets, Clockify is used to generate these timesheets to calculate the amount to invoice each period.
+
+### Generating timesheets
+
+1. Generate a per-member timesheet in Clockify and save (e.g. [aether_201908_bl](https://clockify.me/bookmarks/5dab94738a17084c0337690d) and [aether_201908_yj](https://clockify.me/bookmarks/5dab94828a17084c03376911))
+1. Generate a team timesheet in Clockify and save (e.g. [aether_201908](https://clockify.me/bookmarks/5dab94a0ae125662897c69b4))
+1. Export team timesheet as PDF to be sent to client, if applicable
+
+This process is done using the Clockify [Summary Report](https://clockify.me/reports/summary). Saved reports are found under [Saved Reports](https://clockify.me/reports/saved).
+
+### Generating invoices
+
+1. Generate an invoice in **Quickbooks Online** based on the pricing structure (e.g. timesheet hours and project rate)
+1. Add operational cost line item, if applicable
+1. Add GST, if applicable
+
+The general guidance is that the client is responsible for paying applicable GST and transaction costs charged by their financial institution, and Hypha absorbs the cost of receiving the money (e.g. cost of receiving a wire).
+
+Invoices (and timesheets, if requested) are sent to clients and are payable in CAD, USD, or EUR. Invoices are due 14 days after the invoice date, after which a 5% late fee will be charged for each additional week.
+
+## Member payroll
+
+Member salaries are paid monthly and vary month-to-month depending on their billable hours of work. At the end of each month, each Member's entitled pay is calculated based on the projects and roles they participate in. After withholding taxes and other applicable deductions (e.g. CPP and EI), the net amount is electronically tranferred to the Member's bank account.
```

Comment:
Sorry, did we lock in monthly? I missed this, but would have advocated for biweekly.

Reply:
yes I also don't recollect this being announced.

Also the above does not use terminology inline with any of the requirements for paying (e.g., pay period, salary vs. wage) and I think we should follow that.

Have you reviewed the ESA around wages and pay? Might be helpful to link to direct sections, for instance we have to pay people within a set date of end of pay period, etc...

Reply:
We should use the correct terminology when we are discussing all of these things.

Are we paying salaries? or wages?
We are implying a monthly pay period, that follows the calender month, can we be explicit?

etc...

why wouldn't use the proper terms?

Reply:
I changed the wage language and left pay period ambiguous, let me know if this looks okay.

Reply:
Can @dcwalk or @patcon resolve this if looks ok?